### PR TITLE
Omps reader

### PR DESCRIPTION
**Pull request by jia-jung**
A reader for OMPS HCHO is added

### CODE REVIEW

File: oisatgmi/reader.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from scipy.io import savemat
 import yaml
 import os
-import h5py
+#import h5py
```
Comment (ahsouri) [inline oisatgmi/reader.py]: please bring this back

File: requirements.txt [hunk 2]
```diff
@@ -16,4 +16,4 @@ PyYAML==6.0
 requests==2.28.1
 scipy==1.11.2
 setuptools==63.4.1
-urllib3==1.26.11
\ No newline at end of file
+urllib3==1.26.11
```
Comment (ahsouri) [inline requirements.txt]: please change this back

File: run/control.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # set your python executable path
```
Comment (ahsouri) [inline run/control.yml]: please git rm control.yml

File: oisatgmi/interpolator.py [hunk 4]
```diff
@@ -6,7 +6,7 @@
 from scipy import signal
 from scipy.interpolate.interpnd import _ndim_coords_from_arrays
 from scipy.spatial import cKDTree
-
+from scipy.io import savemat
```
Comment (ahsouri) [inline oisatgmi/interpolator.py]: please remove this after you've tested something

**Review by ahsouri — CHANGES_REQUESTED** [review]

File: oisatgmi/reader.py (hunk 1 above)
Comment (jia-jung) [reply oisatgmi/reader.py]: okay

File: requirements.txt (hunk 2 above)
Comment (jia-jung) [reply requirements.txt]: okay I did 'git rm requirements.txt'

File: run/control.yml (hunk 3 above)
Comment (jia-jung) [reply run/control.yml]: I did

File: oisatgmi/interpolator.py (hunk 4 above)
Comment (jia-jung) [reply oisatgmi/interpolator.py]: okay~

**Comment by jia-jung** [comment]
I made changes as you advised

**Pull request merged**